pkg/cmd/render/env.go

```diff
@@ -12,6 +15,7 @@ var envVarFns = []envVarFunc{
 	getElectionTimeout,
 	getUnsupportedArch,
 	getEtcdName,
+	getTLSCipherSuites,
```

Comment:
yeah error validation is in another piece of code this will fail render.

https://github.com/openshift/cluster-etcd-operator/blob/b5bfb6f22e7b5cb7dbc72e6346e480f5e4e41879/pkg/cmd/render/env.go#L22-L28